Externals/libusb/config.h.in

```diff
@@ -8,7 +8,7 @@
 #undef ENABLE_DEBUG_LOGGING
 
 /* Define to 1 to enable message logging. */
-#undef ENABLE_LOGGING
+#cmakedefine ENABLE_LOGGING 1
```

Comment:
I don't know if it fixed an issue or will produce one, but logging was enabled here and in Externals/libusb/CMakeLists.txt.

Reply:
That is correct. There was no way to enable logging from CMake before.